purchases/src/main/kotlin/com/revenuecat/purchases/common/Backend.kt

```diff
@@ -524,7 +524,7 @@ internal class Backend(
                 synchronized(this@Backend) {
                     paywallEventsCallbacks.remove(paywallEventRequest.cacheKey)
                 }?.forEach { (_, onErrorHandler) ->
-                    onErrorHandler(error, true)
+                    onErrorHandler(error, false)
```

Comment:
I can't think of any reason why we should consider network errors as synced yeah... Might be worth adding some tests for this but this looks good!